app/javascript/modules/listingDetails/ListingDetailsFeatures.tsx

```diff
@@ -39,6 +39,31 @@ const FeatureItem = ({ content, title, toTranslate }: FeatureItemProps) => {
   )
 }
 
+interface UnitDetailsFeatureItemProps {
+  pricingMatrixUrl: string
+}
+const UnitDetailsFeatureItem = ({ pricingMatrixUrl }: UnitDetailsFeatureItemProps) => {
+  const hyperlink = pricingMatrixUrl
+    ? `<a href='${pricingMatrixUrl}' target="_blank">${t(
+        "listings.features.downloadPriceAndIncomeLimitForEachUnitPdf"
+      )}</a>`
+    : ""
+
+  return (
+    <>
+      <Description
+        term={t("listings.features.unitDetails")}
+        description={hyperlink}
+        markdownProps={{
+          disableParsingRawHTML: false,
+        }}
+        markdown={true}
+      />
+      <Description term="" description="" />
```

Comment:
This is a dummy element to work around some styling behavior for the last `Description` component in a list of components:
https://github.com/bloom-housing/ui-components/blob/a068152fd032a48a0630b4b42d3501c658f0241f/src/text/Description.scss#L49
![Screenshot 2023-06-26 at 2 24 59 PM](https://github.com/SFDigitalServices/sf-dahlia-web/assets/133722877/2c560ff2-5bd6-40e0-a5df-cf326f3895d3)

Is there a better way to do this?